paasta_tools/cli/cmds/logs.py

```diff
@@ -328,16 +372,37 @@ def get_log_reader():
 
 
 class LogReader(object):
-    def __init__(self, **kwargs):
-        pass
+    def __init__(self):
+        # Tailing, i.e actively viewing logs as they come in
+        self.SUPPORTS_TAILING = False
```

Comment:
I mean, the very fact that you're setting them in the constructor means they're not. They're a class level attribute that's overriden for each subclass, right?